index.html

```diff
@@ -39,11 +45,13 @@ <h2>I'am a Software Developer</h2>
             </ul>
         </div>
     </div>
-    <div class="grid-container">
-        <section class="myworks">
+    <section class="myworks">
+        <div class="test">
             <h3>My Recent Works</h3>
-            <div class="firstcard">
-                <img src="imgs/placeholder.svg" alt="placeholder" class="placeholder">
+            <span class="m-hidden line"></span></div>
+        <div class="firstcard">
+            <img src="imgs/placeholder.svg" alt="placeholder" class="placeholder">
+            <section class="desktopcard1">
```

Comment:
- [ ] (Optional) Kindly remove the line below the element with `class="firstcard"` so that it looks like that of Figmas. 🙏 

## Your Design

![image](https://user-images.githubusercontent.com/77106962/144675117-cee63e08-ffee-4c7c-bea3-71bccf4c3a4c.png)

## Figma's Design

![image](https://user-images.githubusercontent.com/77106962/144674999-4039d020-44bd-44ce-a7fa-5e1d330a2acb.png)